packages/turf-standard-deviational-ellipse/index.js

```diff
@@ -0,0 +1,133 @@
+import ellipse from '@turf/ellipse';
+
+/**
+ * Takes a {@link FeatureCollection} and returns a standard deviational ellipse,
+ * also known as a “directional distribution.” The standard deviational ellipse
+ * aims to show the direction and the distribution of a dataset by drawing
+ * an ellipse that contains about one standard deviation’s worth (~ 70%) of the
+ * data.
+ *
+ * This module was created in consultation with the following articles:
+ *
+ * • Robert S. Yuill, “The Standard Deviational Ellipse; An Updated Tool for
+ * Spatial Description,” _Geografiska Annaler_ 53, no. 1 (1971): 28–39,
+ * doi:{@link https://doi.org/10.2307/490885|10.2307/490885}.
+ *
+ * • Paul Hanly Furfey, “A Note on Lefever’s “Standard Deviational Ellipse,”
+ * _American Journal of Sociology_ 33, no. 1 (1927): 94—98,
+ * doi:{@link https://doi.org/10.1086/214336|10.1086/214336}.
+ *
+ * It mirrors the functionality of {@link http://desktop.arcgis.com/en/arcmap/10.3/tools/spatial-statistics-toolbox/directional-distribution.htm|Directional Distribution} in ArcGIS and the {@link http://arken.nmbu.no/~havatv/gis/qgisplugins/SDEllipse/|QGIS Standard Deviational Ellipse Plugin}.
+ *
+ * @name standardDeviationalEllipse
+ * @param {Geometry|FeatureCollection<Point>} points GeoJSON points
+ * @param {Object} [options={}] Optional parameters
+ * @param {string} [options.weight] the property name used to weight the center
+ * @param {number} [options.steps=64] number of steps for the polygon
+ * @param {Object} [options.properties={}] properties to pass to the resulting ellipse
+ * @returns {Feature<Polygon>} an elliptical Polygon that includes approximately 1 SD of the dataset within it.
+ * @example
+ *
+ * var bbox = [-74, 40.72, -73.98, 40.74];
+ * var points = turf.randomPoint(400, {bbox: bbox});
+ * var sdEllipse = turf.standardDeviationalEllipse(points, options);
+ *
+ * //addToMap
+ * var addToMap = [points, sdEllipse];
+ *
+ */
+function standardDeviationalEllipse(points, options) {
+    // Optional params
+    options = options || {};
+    if (!isObject(options)) throw new Error('options is invalid');
+    var steps = options.steps || 64;
+    var weightTerm = options.weight;
+    var properties = options.properties || {};
+
+    // Validation:
+    if (!isNumber(steps)) throw new Error('steps must be a number');
+    if (!isObject(properties)) throw new Error('properties must be a number');
+
+    // Calculate mean center & number of features:
+    var numberOfFeatures = coordAll(points).length;
+    var meanCenter = centerMean(points, {weight: weightTerm});
+
+    // Calculate angle of rotation:
+    // [X, Y] = mean center of all [x, y].
+    // theta = arctan( (A + B) / C )
+    // A = sum((x - X)^2) - sum((y - Y)^2)
+    // B = sqrt(A^2 + 4(sum((x - X)(y - Y))^2))
+    // C = 2(sum((x - X)(y - Y)))
+
+    var xDeviationSquaredSum = 0;
+    var yDeviationSquaredSum = 0;
+    var xyDeviationSum = 0;
+
+    featureEach(points, function (point) {
```

Comment:
Might want to use `coordEach` if you plan on supporting `Geometry<Point>` (or drop support single point input).